### ISSUE TITLE
Version tag not populating

### ISSUE THREAD

**Issue by humangenome** [johnnyguitarFP/deadmatterdedisetup#14]
In the Steam query output, we have the Version tag, it currently is showing:

[Version] => 1.0.0.0

Is there a way to get this populating the active server version?

**Comment by johnnyguitarFP** [comment]
No, I believe we need to manually update this. I’ll submit this to the team for review. Thanks!
